src/pages/DetailsPage.js

```diff
@@ -108,6 +108,10 @@ class DetailsPage extends React.PureComponent {
             pronouns = this.props.translate(`pronouns.${localeKey}`);
         }
 
+        const phoneNumber = getPhoneNumber(details);
+        const displayName = isSMSLogin ? this.props.formatPhoneNumber(phoneNumber) : details.displayName;
```

Comment:
I came across this style guide on formatting phone numbers https://developers.google.com/style/phone-numbers#phone-numbers-america

I wonder why the library is formatting the number as  `+34 123412345` (i.e. without space). 
expected:  `+34 123 412 345`.


<img width="386" alt="image" src="https://user-images.githubusercontent.com/29673073/230801598-2b4355c1-fceb-4885-bca0-aaee3bde67b1.png">

Reply:
Alright, I think it treats valid numbers differently. Nothing to do here ✅ 

<img width="391" alt="image" src="https://user-images.githubusercontent.com/29673073/230803531-ffe596a9-ff66-4003-8271-9e6eafe21673.png">